lifecycle-operator/controllers/common/evaluationhandler.go

```diff
@@ -20,8 +20,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+type EvaluationHandlerInterface interface {
+	ReconcileEvaluations(ctx context.Context, phaseCtx context.Context, reconcileObject client.Object, evaluationCreateAttributes CreateEvaluationAttributes) ([]klcv1alpha3.ItemStatus, apicommon.StatusSummary, error)
+	CreateKeptnEvaluation(ctx context.Context, namespace string, reconcileObject client.Object, evaluationCreateAttributes CreateEvaluationAttributes) (string, error)
+}
+
 type EvaluationHandler struct {
-	client.Client
+	Client      client.Client
```

Comment:
we don't need to expose this
```suggestion
	client      client.Client
```

Reply:
When changed to lowercase other function are referencing it with uppercase. so i returned it back to uppercase